admin/controllers/group.js

```diff
@@ -110,18 +121,19 @@ const addGroup = async (req, res, next) => {
       action: 'Add',
       group: req.body,
       validation: validationError.errors,
-      pupilsList: pupilsList
+      pupilsList,
+      selectedPupils
     })
   }
 
   try {
-    group = await groupDataService.create(group)
+    group = await groupService.create({'name': group.name}, group.pupils)
```

Comment:
Unless there are other parameters that can be passed in param 1 here I would just supply name as param 1 as a string.